criminalintentsample/src/main/java/com/bignerdranch/android/criminalintent/CrimeLab.java

```diff
@@ -20,11 +21,20 @@ public static CrimeLab get(Context context) {
 
     private CrimeLab(Context context) {
         mCrimes = new ArrayList<>();
+        Random ran = new Random();
         for (int i = 0; i < 100; i++) {
-            Crime crime = new Crime();
-            crime.setTitle("Crime #" + i);
-            crime.setSolved(i % 2 == 0);
-            mCrimes.add(crime);
+            if(i % (ran.nextInt(3) + 1) == 0) {
+                Crime crime = new Crime();
+                crime.setTitle("Crime #" + i);
+                crime.setSolved(i % 2 == 0);
+                mCrimes.add(crime);
+            } else {
+                Felony crime = new Felony();
+                crime.setTitle("Felony #" + i);
+                crime.setSolved(i % 2 == 0);
+                crime.setDetail("Details");
+                mCrimes.add(crime);
+            }
```

Comment:
I don't quite get what this class is doing with the random. is this generating ID's? Some comments or javadoc would be helpful.